### DISCUSSION TITLE
How to execute renovate in scan mode against local directory?

### DISCUSSION THREAD

**Discussion by rgl** [renovatebot/renovate#18281] [Request Help] [answered]
Is there a way to execute renovate against a local directory without all the bot/repo/git/pr/token workflow?

The idea would be to have something like a scan workflow that would just produce some-kind of json report with the identified dependencies and optionally the available updates.

**Comment by RahulGautamSingh** [comment]
https://docs.renovatebot.com/self-hosted-configuration/#dryrun
Is this what you are looking for? I think the `lookup` option matches your need

**Reply by rgl** [reply]
That looks pretty good!

Does it still require a repository/token/authentication?

I do not really want to integrate this with any repository. I'm just looking for a pure scan mode.

**Comment by rarkins** [comment, marked as answer]
Unsupported. Here is the feature request: #3609
